gulpfile.js

```diff
@@ -67,6 +67,11 @@ gulp.task('styl', () => {
 		.pipe(gulp.dest('dist/assets/css'));
 });
 
+gulp.task('fonts', () => {
```

Comment:
Could we merge this into images and create a `binaries` task? both the fonts and img folder won't change  frequently and adding a task for this seems a bit extra

Reply:
`img` tasks has some more processing in it.. I can rename the task but I'm not sure if we should pass all the binaries through `gulp-responsive`.

Reply:
Since the images are static, I think we should optimize it once and just use it, rather than pointlessly reoptimizing it. We can always use blame / history to get previous versions of the one image in there 😉